GoogleUtilities/Example/GoogleUtilities.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,8 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		40C06D7323AAABE40055BC8B /* GULSceneDelegateSwizzlerTest.m in Sources */ = {isa = PBXBuildFile; fileRef = 40C06D7223AAABE40055BC8B /* GULSceneDelegateSwizzlerTest.m */; };
```

Comment:
We can delete/stop maintaining this file in favor of pure `pod gen` based development.

Reply:
You mean the whole project.pbxproj file? Maybe do it in a separate pr.

Reply:
Yep. That's fine. I wanted to let you know not to go to too much trouble to keep it working.